src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -28,6 +28,7 @@ const AssistantHelper = createReactClass({
       // rather than have both parent and child subscribe to GuideStore.
       currentGuide: null,
       currentStep: 0,
+      currentOrg: null,
```

Comment:
You have currentOrg here but currentOrgSlug elsewhere. Why not just use currentOrg? That way you also don't have to worry about the org slug changing

Reply:
Fixed.